Classes/Backend/Form/FormDataProvider/PageTsConfigFix.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace B13\Pagetsconfig\Backend\Form\FormDataProvider;
+
+/*
+ * This file is part of TYPO3 CMS-extension pagetsconfig by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+use TYPO3\CMS\Backend\Form\FormDataProviderInterface;
+use TYPO3\CMS\Core\Utility\ArrayUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class PageTsConfigFix implements FormDataProviderInterface
+{
+    /**
+     * Add page TsConfig
+     *
+     * @param array $result
+     * @return array
+     */
+    public function addData(array $result)
+    {
+        if (isset($result['pageTsConfig']['TCEFORM.'])) {
+            $tables = $result['pageTsConfig']['TCEFORM.'];
+            foreach ($tables as $tableName => $table) {
+                if (is_array($table) && $result['tableName'].'.' === $tableName) {
+                    if ($table['palettes.']) {
```

Comment:
This is not PHP 8.1 compatible. use `if(is_array($table['palettes.'] ?? null)) {`